src/config.ts

```diff
@@ -7,29 +8,40 @@ const CONFIG_FILE_NAMES = [
   '.soycriticrc.json',
 ];
 
+type CallToImportConfig = {
```

Comment:
Generally, it's better to prefer interfaces over type alias for this kind of thing.